dojo/tools/veracode/parser.py

```diff
@@ -202,13 +202,13 @@ def _get_cwe(val):
             return None
 
     @classmethod
-    def __xml_sca_flaw_to_finding(cls, test, report_date, vendor, library, version, xml_node):
+    def __xml_sca_flaw_to_finding(cls, test, report_date, vendor, library, version, xml_node, dupe_key):
         # Defaults
         finding = Finding()
         finding.test = test
         finding.static_finding = True
         finding.dynamic_finding = False
-        finding.unique_id_from_tool = xml_node.attrib['cve_id']
+        finding.unique_id_from_tool = dupe_key
```

Comment:
This is not a good way to use this field. The id here should come directly from the tool and shouldn't be generated. To fix deduplication we should calculate a meaningful hash_code with and use that for deduplication in `settings.dist.py`.